scripts/pi-hole/php/header.php

```diff
@@ -582,21 +588,21 @@ function pidofFTL()
                 if(strlen($pwhash) > 0 && !$auth) { ?>
                 <li<?php if($scriptname === "login"){ ?> class="active"<?php } ?>>
                     <a href="index.php?login">
-                        <i class="fa fa-user"></i> <span>Login</span>
+                        <i class="fa far fa-user"></i> <span>Login</span>
                     </a>
                 </li>
                 <?php } ?>
                 <!-- Donate -->
                 <li>
                     <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=3J2L3Z4DHW9UY" target="_blank">
-                        <i class="fa fa-paypal"></i> <span>Donate</span>
+                        <i class="fa-icon fab fa-paypal"></i> <span>Donate</span>
```

Comment:
The icon is not centered correctly:
![image](https://user-images.githubusercontent.com/4417660/50748452-59eb6b00-11ee-11e9-9826-967c7246b629.png)